reference/checkouts.sf.yml

```diff
@@ -2626,743 +1363,8 @@ components:
-                              type: array
-                              description: List of discounts applied to this item, as an array of AppliedDiscount objects.
-                              items:
-                                title: Applied Discount
-                                type: object
-                                properties:
-                                  name:
-                                    type: string
-                                    description: The name provided by the merchant.
-                                  discountedAmount:
-                                    type: number
-                                    description: The discounted amount applied within a given context.
-                                    format: double
-                            discountAmount:
-                              type: number
-                              description: The total value of all discounts applied to this item (excluding coupon).
-                              format: double
-                            couponAmount:
-                              type: number
-                              description: The total value of all coupons applied to this item.
-                              format: double
-                            originalPrice:
-                              type: number
-                              description: The item’s original price is the same as the product’s default price.
-                            listPrice:
-                              type: number
-                              description: The item’s list price, as quoted by the manufacturer or distributor.
-                              format: double
-                            salePrice:
-                              type: number
-                              description: The itemʼs price after all discounts are applied. (The final price before tax calculation.)
-                              format: double
-                            extendedListPrice:
-                              type: number
-                              description: The itemʼs list price multiplied by the quantity.
-                              format: double
-                            extendedSalePrice:
-                              type: number
-                              description: The itemʼs sale price multiplied by the quantity.
-                              format: double
-                            type:
-                              type: string
-                              description: the product type - physical or digital
-                            isMutable:
-                              type: boolean
-                              description: ''
-                            isShippingRequired:
-                              type: boolean
-                              description: Whether this item requires shipping to a physical address.
-                            downloadFileUrls:
```

Comment:
Is this field deprecated? I can mark it as deprecated instead of removing it

Reply:
It's still being used in our API and not marked as deprecated 🤔 
I would say remove the `deprecated` and leave it in the documentation please.

Reply:
@valentindellangela Just to clarify, I was referring to `downloadFileUrls` and not `isShippingRequired`. You were also referring to `downloadFileUrls`?

Reply:
oh yes sorry, we have removed this because we can only access the link when we have successfully created an order. Please remove.